pkg/certgen/ca.go

```diff
@@ -16,6 +16,8 @@ var (
 	errNoCAKey  = errors.New("no CA key in file map")
 )
 
+const caCertExpiry = "43800h" // 5 years
```

Comment:
Maybe the bot has a good point? How about making this `const caCertExpiry = 5 * 365 * 24 * time.Hour` and convert to string in `GenerateCA`? This would allow us to drop the comment.

Reply:
Done.